course/sitemaps.py

```diff
@@ -18,33 +17,29 @@ def items(self):
     def location(self, item):
         return reverse(item)
 
+    def priority(self, item):
+        return self.base_priority
 
-class InstanceSitemap(sitemaps.Sitemap):
-    priority = 1.0
+
+class InstanceSitemap(AplusSitemap):
```

Comment:
I think we could set `base_priority = 0.4` for course instances because the course front pages are not as interesting as the actual course contents.